.github/workflows/build-test-linux-x86_64.yml

```diff
@@ -337,6 +337,39 @@ jobs:
         python -m pytest -ra -n 4 --junitxml=${RUNNER_TEST_RESULTS_DIR}/tests_py_core_test_results.xml .
         popd
 
+  tests-py-distributed:
```

Comment:
Shouldnt there be tests for sbsa? even though we dont run them right now, it should be there for when we get a sbsa runner